Rakefile

```diff
@@ -644,21 +644,29 @@ private
 
 ## Before you start...
 
-We will be installing multiple applications.  The installation steps will be provided for each application.
+PLEASE READ THIS WHOLE PAGE! You're going to be installing a lot of stuff on your computer in a way you probably never have before. It's perfectly safe, but it's going to look scary!
```

Comment:
I don't like saying it is scary.  For many, it isn't.  Until you tell them it is.